Content.Client/Atmos/Monitor/UI/Widgets/ThresholdControl.xaml.cs

```diff
@@ -38,106 +30,111 @@ public ThresholdControl(string name, AtmosAlarmThreshold threshold, AtmosMonitor
         {
-            _threshold.SetLimit(AtmosMonitorLimitType.LowerWarning, value);
+            alarmThreshold.SetLimit(AtmosMonitorLimitType.LowerWarning, value);
         };
         _lowerWarningBoundControl.OnBoundEnabled += (isEnabled) =>
         {
-            _threshold.SetEnabled(AtmosMonitorLimitType.LowerWarning, isEnabled);
+            alarmThreshold.SetEnabled(AtmosMonitorLimitType.LowerWarning, isEnabled);
         };
         _lowerWarningBoundControl.OnValidBoundChanged += () =>
         {
-            ThresholdDataChanged!.Invoke(_type, _threshold, _gas);
+            ThresholdDataChanged!.Invoke(thresholdType, alarmThreshold, gasType);
         };
 
-        _warningBounds.AddChild(_lowerWarningBoundControl);
+        WarningBounds.AddChild(_lowerWarningBoundControl);
 
-        _enabled.OnToggled += args =>
+        Enabled.OnToggled += args =>
         {
-            _threshold.Ignore = !args.Pressed;
-            ThresholdDataChanged!.Invoke(_type, _threshold, _gas);
+            alarmThreshold.Ignore = !args.Pressed;
+            ThresholdDataChanged!.Invoke(thresholdType, alarmThreshold, gasType);
         };
-        _enabled.Pressed = !_threshold.Ignore;
+        Enabled.Pressed = !alarmThreshold.Ignore;
     }
 
-    private String LabelForBound(string boundType) //<todo.eoin Replace this with enums
+    private string LabelForBound(string boundType) //<todo.eoin Replace this with enums
     {
         return Loc.GetString($"air-alarm-ui-thresholds-{boundType}");
     }
 
     public void UpdateThresholdData(AtmosAlarmThreshold threshold, float currentAmount)
     {
-        threshold.CheckThreshold(currentAmount, out AtmosAlarmType alarm, out AtmosMonitorThresholdBound which);
+        threshold.CheckThreshold(currentAmount, out var alarm, out var which);
 
         var upperDangerState = AtmosAlarmType.Normal;
         var lowerDangerState = AtmosAlarmType.Normal;
         var upperWarningState = AtmosAlarmType.Normal;
         var lowerWarningState = AtmosAlarmType.Normal;
 
-        if(alarm == AtmosAlarmType.Danger)
-        {
-            if(which == AtmosMonitorThresholdBound.Upper) upperDangerState = alarm;
-            else lowerDangerState = alarm;
-        }
-        else if(alarm == AtmosAlarmType.Warning)
+        switch (alarm)
         {
-            if(which == AtmosMonitorThresholdBound.Upper) upperWarningState = alarm;
-            else lowerWarningState = alarm;
+            case AtmosAlarmType.Danger when which == AtmosMonitorThresholdBound.Upper:
+                upperDangerState = alarm;
+                break;
+            case AtmosAlarmType.Danger:
+                lowerDangerState = alarm;
+                break;
+            case AtmosAlarmType.Warning when which == AtmosMonitorThresholdBound.Upper:
+                upperWarningState = alarm;
+                break;
+            case AtmosAlarmType.Warning:
+                lowerWarningState = alarm;
+                break;
```

Comment:
Is the switch statement easier to understand vs this fancy switch/case/when statement?